### PR TITLE
Revisions for Chapter 1 (First two paragraphs)

### PR DESCRIPTION
**Pull request by kangqiao-ctrl**
第一章，第一、二小节：
- 更正了个别金融、心理学等专有名词的翻译。
- 修正了一些俚语表达，使之更贴近原文意思。
- 调整了一些句子的语序同时增减了个别句子的成分。
- 增加了个别注释和维基词条，增加了中文版的可读性。
- 修正了个别称谓和语气词，使正文更贴近原文意思。

### CODE REVIEW

File: chapters/01 第一章-因果关系入门.ipynb [hunk 1]
```diff
@@ -6,42 +6,42 @@
...
     "\n",
-    "ML 在这种逆因果关系类型的问题上是出了名的糟糕。这类问题要求我们回答“假设发生”这样的问题，经济学家称之为反事实。假设我目前要求的商品不是这个价格，而是使用另一个价格，会发生什么情况？假设我不采用这种低脂饮食，而是采用低糖饮食，会发生什么？假设您在银行工作，提供信贷，您将必须弄清楚更改客户线会如何改变您的收入。或者，假设您在当地政府工作，您可能会被要求弄清楚如何改善学校教育系统。您是否应该因为数字知识时代告诉您而将平板电脑送给每个孩子？或者你应该建造一个老式的图书馆？\n",
+    "ML 在这种逆因果关系类型的问题上是出了名的糟糕。这类关系要求我们回答“假设发生”之类的问题，经济学家们称之为反事实推理。假设我用把商品现价换成另一个价格，而不是让商品继续保持现在的要价不变，会发生什么情况？假设我不再继续当前的低脂饮食，而是采用低糖饮食，会发生什么？假设你在银行工作，负责提供信贷，你将必须弄明白更改客户信用额度会对银行收入产生什么影响。或者说，假设你在当地政府工作，你可能会被要求搞清楚怎么能改善一下学校教育系统。你要不要仅仅凭着“数字知识时代的常识”就给每个孩子一人一台平板电脑，还是说，你应该建一个老式的图书馆？\n",
     "\n",
-    "这些问题的核心是我们希望知道答案的因果调查。因果问题渗透到日常问题中，例如弄清楚如何提高销售额，但它们也在我们非常个人和宝贵的困境中发挥重要作用：我是否必须上一所昂贵的学校才能在生活中取得成功（是吗？教育导致收入）？移民是否会降低我找到工作的机会（移民是否会导致失业率上升）？向穷人汇款会降低犯罪率吗？不管你在哪个领域，很可能你已经或将不得不回答某种类型的因果问题。不幸的是，对于 ML，我们不能依靠相关类型预测来解决它们。\n",
+    "每一个这样的问题中心都有一个因果性的疑惑等着我们去解答。因果问题贯穿在日常生活中，诸如“去研究一下如何提高销售额”之类的问题里。但当我们面对极为个人且重要的抉择时，因果问题同样重要：我是否必须上一所昂贵的学校才能在生活中取得成功（受教育程度影响收入吗）？移民是否会降低我找到工作的机会（移民是否会导致失业率上升）？面向穷人的社会救助手段（money transfer，低保资金转移）会降低犯罪率吗？不管你在哪个领域，你都很有可能即将，或者曾经，不可避免地要回答某种形式的因果问题。不幸的是，对于机器学习而言，这些问题不能依靠基于相关性的预测来解决。\n",
     "\n",
-    "回答这类问题比大多数人想象的要困难。您的父母可能已经向您反复说过“关联不是因果关系”，“关联不是因果关系”。但实际上要解释为什么会这样。这就是因果推理简介的全部内容。至于本书的其余部分，它将致力于弄清楚如何使关联成为因果关系。\n",
```
Comment (xieliaing) [inline chapters/01 第一章-因果关系入门.ipynb:44]: I think “parents ” here should still be interpreted as “父母”, to emphasize that association is not causation should be a well known knowledge in our early learning experiece.

Comment (kangqiao-ctrl) [reply chapters/01 第一章-因果关系入门.ipynb:44]: That makes total sense. Let's keep using "父母" under this context.

**Review by kangqiao-ctrl — COMMENTED** [review]
- Minor wording changes

**Review by kangqiao-ctrl — COMMENTED** [review]
- Switched back to '父母' for the English word 'parents'.

**Pull request merged**